app/controllers/application_controller.rb

```diff
@@ -83,15 +83,25 @@ def filter_params
   end
 
   def check_accounts_enabled
-    unless accounts_enabled?
+    if Rails.configuration.feature_flag_govuk_accounts
```

Comment:
Use `account_feature_flag_enabled?` here too for consistency

Reply:
quite right, that should have been there!

Reply:
amended here: https://github.com/alphagov/finder-frontend/pull/2389/commits/4631afc5bb629ccd7149055b8a387fcf7d9a1785